ams/ams-server/src/main/java/com/netease/arctic/ams/server/controller/LoginController.java

```diff
@@ -46,12 +47,17 @@ public static void getCurrent(Context ctx) {
   public static void login(Context ctx) {
     // ok
     JSONObject postBody = ctx.bodyAsClass(JSONObject.class);
-    if (postBody.get("user").equals("admin") && (postBody.get("password").equals("admin"))) {
-      ctx.sessionAttribute("user", new SessionInfo("admin", System.currentTimeMillis() + ""));
+    if (postBody.get("user").equals(ArcticMetaStore.conf.get(ArcticMetaStoreConf.LOGIN_USERNAME)) && (postBody.get(
```

Comment:
May `ArcticMetaStore.conf.get(ArcticMetaStoreConf.LOGIN_USERNAME).equals(postBody.get("user"))  && ArcticMetaStore.conf.get(ArcticMetaStoreConf.LOGIN_PASSWORD).equals(postBody.get("password")) ` be better in case of  the expression `postBody.get("xxx")` returns null  .